### PR TITLE
chore: upgrade sequelize and sequelize-typescript version.

### LINKED ISSUE #34: how long to start support v6
how long to start support v6

### PR DESCRIPTION
**Pull request by stu01509**
Fix #34 

In this PR, I updated the sequelize version to `6.3.5`, but sequelize-typescript support sequelize v6 version still in beta.

cc @hsluoyz

### CODE REVIEW

File: src/adapter.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { ModelCtor } from 'sequelize-typescript';
```
Comment (hsluoyz) [inline src/adapter.ts]: ???

**Comment by hsluoyz** [comment]
Don't use two commits.

**Comment by hsluoyz** [comment]
@nodece please review.

@stu01509 what I'm worried about is the code quality. Two places need to be improved in such a simple PR.

File: src/adapter.ts (hunk 1 above)
Comment (stu01509) [reply src/adapter.ts]: I don't notice, fixed it.

**Comment by hsluoyz** [comment]
Plz fix the CI.

**Comment by nodece** [comment]
@stu01509 What is the reason for closing?

**Comment by nodece** [comment]
I think it's enough to using `sequelize` .

**Pull request closed without merging**